cnames_active.js

```diff
@@ -2350,6 +2350,7 @@ var cnames_active = {
   "spin": "fgnass.github.io/spin.js", // noCF? (don´t add this in a new PR)
   "spirare": "waldenn.github.io/spirare", // noCF? (don´t add this in a new PR)
   "spiritual": "spiritual-js.github.io", // noCF? (don´t add this in a new PR)
+  "splash": "bejoistic.github.io",
```

Comment:
```suggestion
  "splash": "bejoistic.github.io/SplashJS",
```
If you are wanting to target https://github.com/bejoistic/SplashJS, this should be updated to reflect that

Reply:
Oh sorry, fixed now